src/snipsonian/observable-state/src/actionableStore/entities/asyncEntityActionCreators.ts

```diff
@@ -37,6 +39,13 @@ export interface IAsyncEntityActionCreators<ActionType, State, ExtraProcessInput
         props: ICreateFetchAsyncEntityActionProps<State, StateChangeNotificationKey, ExtraInput, ApiInput, ApiResult, ApiResponse>
         // eslint-disable-next-line max-len
     ): IObservableStateAction<ActionType, IAsyncEntityActionPayload, State, ExtraProcessInput, StateChangeNotificationKey>;
+
+    downloadAsyncEntityAction<ExtraInput extends object, ApiInput, ApiResult, ApiResponse = ApiResult>(
```

Comment:
I don't think you need to create an entire new asyncEntityAction for the download, can't you just use the fetch for this?